### PR TITLE
docs: openssh updates

### PR DESCRIPTION
**Pull request by stevenGravy**
- include `sudo` as that's typically required in the environment
- Show labels
- Describe how to edit the node after creation

### CODE REVIEW

File: docs/pages/server-access/guides/openssh.mdx
Comment (zmb3) [inline docs/pages/server-access/guides/openssh.mdx:64]:
```suggestion
$ sudo teleport join openssh \
  --address <Var name="node" /> \
  --proxy-server <Var name="proxy" /> \
```

I would break lines for each flag, rather than having 2 flags on the first line and one flag on all following lines

File: docs/pages/server-access/guides/openssh.mdx
Comment (zmb3) [reply docs/pages/server-access/guides/openssh.mdx:1]: No, good find!

Comment (stevenGravy) [reply docs/pages/server-access/guides/openssh.mdx:1]: Yes that looks incorrect, please look update that is inline with other examples.

File: docs/pages/server-access/guides/openssh.mdx
Comment (stevenGravy) [reply docs/pages/server-access/guides/openssh.mdx:73]: Yes, I've updated to that type of example. Also added token variable.

File: docs/pages/server-access/guides/openssh.mdx
Comment (alexfornuto) [inline docs/pages/server-access/guides/openssh.mdx:62]:
```suggestion
resource on your cluster with the following command, assigning
<Var name="teleport.example.com:443" /> to the address and port of your Teleport
Proxy Service, <Var name="server1.example.com:22" /> to the address and port of
the node that will join the cluster, and <Var name="token" /> to the token value:
```

Adjusted to mention port as well as address, and for line breaks ~80 chars.

**Review by alexfornuto — COMMENTED** [review]
LGTM pending outstanding comment.

**Comment by stevenGravy** [comment]
> LGTM pending outstanding comment.

done, thanks @alexfornuto !

**Review by alexfornuto — APPROVED** [review]

File: docs/pages/server-access/guides/openssh.mdx
Comment (lsgunn-teleport) [inline docs/pages/server-access/guides/openssh.mdx:51]:
```suggestion
To register the OpenSSH node, you must have a valid auth token to connect to the cluster.
```

File: docs/pages/server-access/guides/openssh.mdx
Comment (lsgunn-teleport) [inline docs/pages/server-access/guides/openssh.mdx:52]:
```suggestion
You can generate the token by running the following command against your Teleport Auth Service:
```

File: docs/pages/server-access/guides/openssh.mdx
Comment (lsgunn-teleport) [inline docs/pages/server-access/guides/openssh.mdx:60]: I might suggest splitting this into a statement and list. For example:
Copy the `teleport` binary to your `sshd` host. Configure `sshd` and create a node 
resource on your cluster with the following command:

```
command
```

Change the command-line options to assign the following values:
- Set <Var name="teleport.example.com:443" /> to the address and port of your Teleport
Proxy Service.
- Set <Var name="server1.example.com:22" /> to the address and port of
the node that will join the cluster.
- Set <Var name="token" /> to the token value.

File: docs/pages/server-access/guides/openssh.mdx
Comment (lsgunn-teleport) [inline docs/pages/server-access/guides/openssh.mdx:76]:
```suggestion
hostname and labels with `tctl edit nodes/<hostname>`.  If the hostname isn't unique, get the UUID
```

**Review by lsgunn-teleport — APPROVED** [review]
A couple of minor suggestions.

**Comment by alexfornuto** [comment]
@stevenGravy @lsgunn-teleport's suggestions are definitely improvements for readability, I suggest reviewing her suggestions and incorporating before proceeding with the backport.

**Comment by stevenGravy** [comment]
> @stevenGravy @lsgunn-teleport's suggestions are definitely improvements for readability, I suggest reviewing her suggestions and incorporating before proceeding with the backport.

I've opened a PR with those edits and will include the backport after approved.

**Pull request merged**
